personal_assistant/main.py

```diff
@@ -78,6 +78,38 @@ def main():
         elif command == "entries":
             print(num_records(personal_assistant))
 
+        elif command == "add-note":
+            personal_assistant.add_note()
+
+        elif command == "remove-note":
+            note_name = input("Enter the name of the note to delete: ")
+            personal_assistant.remove_note(note_name)
+            print("Note deleted successfully.")
```

Comment:
I think the this message needs to be shown only when a note was really deleted:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/d8c27c6b-c72a-4acb-927a-11298a4858ca)

Reply:
+